src/ui_parts/shortcut_panel.tscn

```diff
@@ -0,0 +1,57 @@
+[gd_scene load_steps=4 format=3 uid="uid://jf1wq6xy617n"]
+
+[ext_resource type="Script" path="res://src/ui_parts/shortcut_panel.gd" id="1_bdakw"]
+[ext_resource type="Texture2D" uid="uid://gxcy17tjtyfv" path="res://visual/icons/DragHandle.svg" id="2_8txvx"]
+[ext_resource type="Texture2D" uid="uid://ckkkgof1hcbld" path="res://visual/icons/Gear.svg" id="3_lpgyc"]
+
+[node name="ShortcutPanel" type="VBoxContainer"]
+custom_minimum_size = Vector2(100, 0)
+anchors_preset = 7
+anchor_left = 0.5
+anchor_top = 1.0
+anchor_right = 0.5
+anchor_bottom = 1.0
+offset_left = -62.0
+offset_top = -190.0
+offset_right = 38.0
+offset_bottom = -140.0
+grow_horizontal = 2
+grow_vertical = 0
+script = ExtResource("1_bdakw")
+
+[node name="PanelContainer" type="PanelContainer" parent="."]
+custom_minimum_size = Vector2(0, 50)
+layout_mode = 2
+theme_type_variation = &"DarkPanel"
+metadata/_edit_use_anchors_ = true
+
+[node name="HBoxContainer" type="HBoxContainer" parent="PanelContainer"]
+layout_mode = 2
+theme_override_constants/separation = 8
+alignment = 1
+
+[node name="DragHandle" type="TextureRect" parent="PanelContainer/HBoxContainer"]
+custom_minimum_size = Vector2(35, 0)
+layout_mode = 2
+mouse_default_cursor_shape = 6
+texture = ExtResource("2_8txvx")
+stretch_mode = 3
```

Comment:
I think we should have a structure like this:

PanelContainer
- Dots
- MarginContainer
  - HBox
    - Buttons

The panel container could be edited from its theme variations to have smaller paddings (look for example at main_scene.tscn the update_theme() method). On top, left, and right of the buttons, the padding should be the same.